WhisperAPI/WhisperAPI/Services/SuggestionsService.cs

```diff
@@ -103,8 +104,7 @@ private IEnumerable<SuggestedDocument> SearchCoveoIndex(string query)
         private bool IsIntentRelevant(NlpAnalysis nlpAnalysis)
```

Comment:
Add unit tests for this. You can either make this method public or test the already public method "IsQueryRelevant" by mocking the nlp call. I want to see tests with and without the wild card to see that everything works as expected.